tests/providers/microsoft/azure/hooks/test_azure_container_instance.py

```diff
@@ -52,7 +52,10 @@ def setup_test_cases(self, create_mock_connection):
                 conn_type="azure_container_instances",
                 login="login",
                 password="key",
-                extra={"tenantId": "tenant_id", "subscriptionId": "subscription_id"},
+                extra={
+                    "tenantId": "63e85d06-62e4-11ee-8c99-0242ac120002",
```

Comment:
Is it a valid id? or just some random one?

Reply:
it is Random. I use https://www.uuidgenerator.net/ to generate it. The new sdk throws a value error if tenantId is not valid uuid